pkg/datapath/maps/map.go

```diff
@@ -175,6 +175,22 @@ func (ms *MapSweeper) RemoveDisabledMaps() {
 		}...)
 	}
 
+	if !option.Config.EnableBPFMasquerade {
+		maps = append(maps, []string{"cilium_snat_v4_external"}...)
+	}
+
+	if !option.Config.EnableIPv4Masquerade {
+		maps = append(maps, []string{"cilium_snat_v4_external"}...)
+	}
```

Comment:
This will attempt to remove the map twice if both `EnableBPFMasquerade` and `EnableIPv4Masquerade` are false. I think it can be merge into a single if condition. The same is true with the if statements below. Could you merge them?

Reply:
> This will attempt to remove the map twice if both EnableBPFMasquerade and EnableIPv4Masquerade are false

that's true but I thought this won't be a problem because the function does not error out if the path of the map is not found in the filesystem.

Reply:
I wired the requested changes. It should be more concise. thanks @pchaigno